CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 ## Upcoming Release
 
 ### Added
+
+- **NSUbiquitousKeyValueStore**:
+    -Added `subscript(key)`, 
```

Comment:
I think it's ok to remove the new lines for each one.

Reply:
Do you mean the newlines between each new function, and put them all on one line?  I was just trying to match the previous change notes here.

Reply:
Most of the time they're on 1 line. I think it's better that way.